internal/service/connect/hours_of_operation.go

```diff
@@ -0,0 +1,336 @@
+package connect
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/connect"
+	"github.com/hashicorp/aws-sdk-go-base/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	tftags "github.com/hashicorp/terraform-provider-aws/internal/tags"
+)
+
+func ResourceHoursOfOperation() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceHoursOfOperationCreate,
+		ReadContext:   resourceHoursOfOperationRead,
+		UpdateContext: resourceHoursOfOperationUpdate,
+		DeleteContext: resourceHoursOfOperationDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(connectHoursOfOperationCreatedTimeout),
+			Delete: schema.DefaultTimeout(connectHoursOfOperationDeletedTimeout),
+		},
+		Schema: map[string]*schema.Schema{
+			"config": {
```

Comment:
config can have between 0 and 100 items. Can you add validation logic?

Reply:
Resolved in af1e8156408209cc246b110dd3dac257787e239e

Reply:
Hi @drewmullen reference for hard limit retrieved from here